README.md

```diff
@@ -369,21 +369,20 @@ Account created in the previous step at the Org A:
     cd ./quota-monitorings-solution/terraform/example
     ```
 
-### 3.6 Download Service Account Key File
+### 3.6 Set OAuth Token Using Service Account Impersonization
 
-Create Service Account key from host project A. The service account key file
-will be downloaded to your machine as key.json. After you download the key
-file, you cannot download it again.
+Impersonate your host project service account and set environment variable using temporary token to authenticate terraform. You will need to make sure your user has the [Service Account Token Creator role](https://cloud.google.com/iam/docs/service-account-permissions#token-creator-role) to create short-lived credentials.
 
 ```sh
-gcloud iam service-accounts keys create key.json \
-    --iam-account=$SERVICE_ACCOUNT_ID@$DEFAULT_PROJECT_ID.iam.gserviceaccount.com
+gcloud config set auth/impersonate_service_account \
```

Comment:
When testing I didn't have the necessary role, so I got an error like: "unable to complete impersonation".  To fix it I had to add the role, unset the impersonation and try again, so somewhere in here it would be good to have a note that says something like.

"If you get an error saying unable to impersonate, you will need to unset the impersonation. Have the role added and try again."

Reply:
Great point. I can add